### ISSUE TITLE
json-diff depends on broken version (0.6.0) of dreamopt

### ISSUE THREAD

**Issue by mpiroc** [andreyvit/json-diff#49]
TL;DR: json-diff should depend on dreamopt 0.8.0 instead of 0.6.0.

Version 0.6.0 has a [corrupted tarball](https://github.com/andreyvit/dreamopt.js/issues/4). `npm` seems to handle this fine, but other package managers like `pnpm` fail to install it (you can also try extracting it yourself using 7zip, and observing the error).

Because of this, running `pnpm install` on a package that depends on `json-diff` will fail with the following message:

```
ERROR· Unexpected end of data
--
174 | at _final                     ../../../../../../../../../root/.rush/node-v12.13.0/pnpm-4.9.3/node_modules/pnpm/lib/node_modules/tar-stream/extract.js:253                                            if (this._partial) retur…
175 | at callFinal                  ../../../../../../../../../root/.rush/node-v12.13.0/pnpm-4.9.3/node_modules/pnpm/lib/node_modules/tar-stream/node_modules/readable-stream/lib/_stream_writable.js:590  stream._final(function (…
176 | at processTicksAndRejections  internal/process/task_queues.js:80
```

In other words, `pnpm` users are blocked from using `json-diff` until it updates its dependency on cloudopt. The most recent version of cloudopt (0.8.0) has a fixed tarball. I did not test 0.7.0.

**Comment by ewoudenberg** [comment]
Version 0.6.0 updates `dreamopt` to v0.8.0
